src/commands/app/build.js

```diff
@@ -163,8 +166,8 @@ Build.flags = {
     allowNo: true
   }),
   'force-build': Flags.boolean({
```

Comment:
The default behavior of buildActions has changed, it will always build actions that it determines 'need' to be built.  In this case, force-build can still be used if an undetectable change has been made.  For example, if an action imports|requires a module and the root application's package.json dependency version was updated _(and `npm i` was run)_ this change would not be noticed and the actions would not be rebuilt.
Effectively `force-build` could be thought of as clean-and-build